Dockerfile.windows

```diff
@@ -1,17 +1,24 @@
-FROM microsoft/dotnet:1.1-sdk-msbuild
+# escape=`
+FROM microsoft/dotnet:2.0-sdk-nanoserver
```

Comment:
I feared we would hit this. In general, the problem is that the the `dev` branch of this repo builds with a pre-release version of .NET Core that is newer than what the nightly docker images contain. I'm not sure we really have a good solution for this at the moment.